apps/emqx_message_transformation/test/emqx_message_transformation_http_api_SUITE.erl

```diff
@@ -2057,3 +2057,28 @@ t_many_transformations_order(_Config) ->
     Topic = <<"t/a">>,
     ?assertIndexOrder(Names, Topic),
     ok.
+
+%% Original issue: https://emqx.atlassian.net/browse/EMQX-14604
```

Comment:
Nit. Perhaps better to have the issue description here instead of an URL inaccessible w/o credentials?

Reply:
The description was added as comments for the steps to reproduce the issue.